test/encryption_test.rb

```diff
@@ -0,0 +1,54 @@
+require './test/test_helper'
+require './lib/encryption'
+require './lib/shifts'
+
+class EncryptionTest < Minitest::Test
+  def setup
+    @encryption = Encryption.new
+    @shifts = Shifts.new
```

Comment:
Created an instance of shifts here to implement stub testing and to use shifts methods"